alerts/memcached/README.md

```diff
@@ -0,0 +1,40 @@
+
+# Alerts for Memcached in the Ops Agent
+
+## High CPU Usage
+
+if `memcached.cpu.usage` is higher than what is anticipated it could show that the Server us overloaded and efforts to improve performance may be necessary. Subjective to environment, configure threshold knowingly.
```

Comment:
Please capitalize the first letter in the descriptions

Reply:
typo, `us` --> `is`

Reply:
Could you please also include in the description what threshold is being used in the sample alert policy?

Reply:
Fixed